duktape/src/main/jni/DuktapeContext.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2016 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef DUKTAPE_ANDROID_DUKTAPE_CONTEXT_H
+#define DUKTAPE_ANDROID_DUKTAPE_CONTEXT_H
+
+#include <jni.h>
+#include "duktape.h"
+
+class Duktape {
```

Comment:
Tough one. Should this file be called Duktape.h or should this be called DuktapeContext ?